src/nidmm/metadata/functions_addon.py

```diff
@@ -34,31 +34,34 @@
     'GetCalCount':                     { 'codegen_method': 'no',       },  # Calibration function not exposed in Python API
     'FormatMeasAbsolute':              { 'codegen_method': 'no',       },  # Utility function for C customers
     'IsUnderRange':                    { 'codegen_method': 'no',       },  # Utility function for C customers
-    'IsOverRange':                     { 'codegen_method': 'no',       },  # Utility function for C customers
+    'ConfigureThermistorType':         { 'codegen_method': 'no',       },  # Utility function for C customers
+    'ConfigureTransducerType':         { 'codegen_method': 'no',       },  # Attribute already available
```

Comment:
use the same wording we used in NI-DCPower's addon.py

Reply:
Fixed